tools/debian_kernel_postinst_lunatik.sh

```diff
@@ -42,6 +42,7 @@ else
   cd "${LUNATIK_DIR}"
 fi
 make -j"${CPU_CORES}" KERNEL_RELEASE="${KERNEL_RELEASE}" && make KERNEL_RELEASE="${KERNEL_RELEASE}" install || exit 1
+make ipparse_install
```

Comment:
I would make it optional

Reply:
OK so I’ll remove it from the script, as there is no option passing for it.

Reply:
my suggestion is to have an option.. e.g., CONFIG_MOONSTATIK or something..